images/installer/Dockerfile.upi.ci

```diff
@@ -44,6 +44,7 @@ RUN python -m pip install pyopenssl
 ENV CLOUDSDK_PYTHON=/usr/bin/python
 
 RUN python3 -m pip install yq
+RUN curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /bin/yq-3.3.0 && chmod +x /bin/yq-3.3.0
```

Comment:
also change the output file to `/bin/yq`

Reply:
1. In release repo, there is some place downloading v4.13.5, to avoid future conflicts, I named the yq tools with version number.
```
$ grep -r "yq/releases/download" |grep -v 3.3.0
ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml:        curl -L https://github.com/mikefarah/yq/releases/download/2.4.0/yq_linux_amd64 2>/dev/null >/tmp/bin/yq
ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml:        curl -L https://github.com/mikefarah/yq/releases/download/2.4.0/yq_linux_amd64 2>/dev/null >/tmp/bin/yq
ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml:        curl -L https://github.com/mikefarah/yq/releases/download/2.4.0/yq_linux_amd64 2>/dev/null >/tmp/bin/yq
ci-operator/step-registry/assisted/baremetal/operator/publish/assisted-baremetal-operator-publish-commands.sh:curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
ci-operator/config/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master.yaml:      RUN curl -Lk -o /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.18.1/yq_linux_amd64 && chmod +x /usr/bin/yq
```
2. On the line 46, another yq is installed from pip, to avoid confusion, I named it with version number.

Reply:
Of course, we can change the output file to `/bin/yq`, and calling the specific `yq` with full path from the scripts in release repo. That also works for me.

Reply:
Any suggestion?